DQMOffline/Trigger/BuildFile.xml

```diff
@@ -31,4 +31,6 @@
 
 <use   name="root"/>
 <use   name="boost"/>
-<flags   EDM_PLUGIN="1"/>
```

Comment:
This is the change that is breaking the IBs

Reply:
To make things work, DQMOffline/Trigger/src/SealModule.cc would have to move to the plugins directory. Even better, all the modules used in that file would also move to that directory.

Reply:
sorry about that! Fixed now. All plugins are now in plugins dir

Reply:
https://github.com/cms-sw/cmssw/pull/24390, its the same branch with the fix